packages/sdk-core/src/proposals/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 Webb Technologies Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+export const LE = true;
+export const BE = false;
```

Comment:
Is this meant to indicate the proposals should always be encoded as little endian?

Reply:
No, this is a boolean flag used in certain functions to indicate the type of encoding you want. It exists on the `DataView` functions. I will add a comment.